app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -168,25 +169,58 @@ class StatePlayerRecyclerViewAssembler private constructor(
    * configuration of this assembler, as well as the GCS entity ID that should be associated with
    * rich-text rendering for this state.
    */
-  fun compute(ephemeralState: EphemeralState, gcsEntityId: String): List<StateItemViewModel> {
+  fun compute(
+    ephemeralState: EphemeralState,
+    gcsEntityId: String,
+    isSplitView: Boolean
+  ): Pair<List<StateItemViewModel>, List<StateItemViewModel>> {
     val hasPreviousState = ephemeralState.hasPreviousState
 
     previousAnswerViewModels.clear()
-    val pendingItemList = mutableListOf<StateItemViewModel>()
+    val leftPendingItemList = mutableListOf<StateItemViewModel>()
+    val rightPendingItemList = mutableListOf<StateItemViewModel>()
     if (playerFeatureSet.contentSupport) {
-      addContentItem(pendingItemList, ephemeralState, gcsEntityId)
+      addContentItem(leftPendingItemList, ephemeralState, gcsEntityId)
     }
     val interaction = ephemeralState.state.interaction
+
     if (ephemeralState.stateTypeCase == EphemeralState.StateTypeCase.PENDING_STATE) {
-      addPreviousAnswers(pendingItemList, ephemeralState.pendingState.wrongAnswerList, gcsEntityId)
+      addPreviousAnswers(
+        leftPendingItemList,
+        rightPendingItemList,
+        ephemeralState.pendingState.wrongAnswerList,
+        /* isCorrectAnswer= */ false,
+        isSplitView,
+        gcsEntityId
+      )
       if (playerFeatureSet.hintsAndSolutionsSupport) {
         hintHandler.maybeScheduleShowHint(ephemeralState.state, ephemeralState.pendingState)
       }
-      if (playerFeatureSet.interactionSupport) {
-        addInteractionForPendingState(pendingItemList, interaction, hasPreviousState, gcsEntityId)
+      if (playerFeatureSet.interactionSupport && !isSplitView) {
+        addInteractionForPendingState(
+          leftPendingItemList,
+          interaction,
+          hasPreviousState,
+          gcsEntityId
+        )
+      }
+      if (playerFeatureSet.interactionSupport && isSplitView) {
+        addInteractionForPendingState(
```

Comment:
Can this fit on one line? If so, prefer to.

Also, I think we can simplify this slightly:

```kotlin
if (playerFeatureSet.interactionSupport) {
  val interactionItemList = if (isSplitView) rightPendingItemList else leftPendingItemList
  addInteractionForPendingState(interactionItemList, interaction, hasPreviousState, gcsEntityId)
}
```

Reply:
>Can this fit on one line? If so, prefer to.

It will fail in lint checks (Exceeded max line length)

>if (playerFeatureSet.interactionSupport) {
  val interactionItemList = if (isSplitView) rightPendingItemList else leftPendingItemList
  addInteractionForPendingState(interactionItemList, interaction, hasPreviousState, gcsEntityId)
}

Yes, this is better

Reply:
Resolving this on behalf of Ben.